packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-tile.tsx

```diff
@@ -0,0 +1,603 @@
+import {
+  Component,
+  Prop,
+  h,
+  Element,
+  Host,
+  State,
+  Fragment,
+} from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import {
+  OdsChipSize,
+  OdsChipVariant,
+  OdsHTMLAnchorElementTarget,
+  OdsIconName,
+  OdsIconSize,
+} from '@ovhcloud/ods-core';
+import { HTMLStencilElement } from '@stencil/core/internal';
+import {
+  getTranslation,
+  fetchLocaleStringsForComponent,
+} from '../../../../utils/translation.utils';
+import formatDate from '../../../../utils/date.utils';
+import { fetchServiceId, fetchServiceDetails, fetchDomainOwner } from './Api';
+
+export interface IMscBillingTile {
+  offer?: string;
+  dataTracking?: string;
+  language: string;
+  servicePath: string;
+}
+
+@Component({
+  tag: 'msc-billing-tile',
+  styleUrl: 'msc-billing-tile.scss',
+  shadow: true,
+})
+export class MscBillingTile implements IMscBillingTile {
+  @Element() host!: HTMLStencilElement;
+
+  /** Name of the offer */
+  @Prop() public offer?: string = '';
+
+  /** label sent to the tracking service */
+  @Prop() public dataTracking?: string = '';
+
+  /** language of the billing-tile, french default */
+  @Prop() public language = 'fr-FR';
+
+  /** service path for the API */
+  @Prop() public servicePath = '';
+
+  @State() localStrings: { [key: string]: string };
+
+  @State() private tabIndex = 0;
+
+  @State() serviceId: string;
+
+  @State() creationDate: string;
+
+  @State() nextBillingDate: string;
+
+  @State() contactAdmin: string;
+
+  @State() contactBilling: string;
+
+  @State() contactTech: string;
+
+  @State() renewStatus: string;
+
+  @State() commitmentStatus: string;
+
+  @State() requestDate: string;
+
+  @State() whoisOwnerDomain: string;
+
+  async componentWillLoad() {
+    this.localStrings = await fetchLocaleStringsForComponent(this.language);
+    await fetchServiceId(this.servicePath).then((serviceData) => {
+      this.contactAdmin = serviceData.contactAdmin;
+      this.contactBilling = serviceData.contactBilling;
+      this.contactTech = serviceData.contactTech;
+      this.serviceId = serviceData.serviceId;
+      this.creationDate = formatDate(serviceData.creation, this.language);
+      if (serviceData.status === 'ok') {
+        if (serviceData.renew.deleteAtExpiration === true) {
+          // Red chip 'Cancellation requested', link in menu 'Stop cancellation of service'
+          this.renewStatus = 'deleteAtExpiration';
+        } else {
+          // service still active
+          if (
+            serviceData.renew.automatic === true &&
```

Comment:
why not use serviceData.renew.automatic && ! serviceData.renew.manualPayment ???